rotkehlchen/data_import/importers/rotki_trades.py

```diff
@@ -0,0 +1,80 @@
+import csv
+from pathlib import Path
+from typing import Any, Dict
+
+from rotkehlchen.assets.utils import symbol_to_asset_or_token
+from rotkehlchen.data_import.utils import BaseExchangeImporter
+from rotkehlchen.db.drivers.gevent import DBCursor
+from rotkehlchen.errors.asset import UnknownAsset
+from rotkehlchen.errors.misc import InputError
+from rotkehlchen.errors.serialization import DeserializationError
+from rotkehlchen.exchanges.data_structures import Trade
+from rotkehlchen.serialization.deserialize import (
+    deserialize_asset_amount,
+    deserialize_timestamp_from_date,
+)
+from rotkehlchen.types import Fee, Location, Price, TradeType
+
+
+class RotkiGenericTradesImporter(BaseExchangeImporter):
+    def _consume_rotki_trades(
+            self,
+            cursor: DBCursor,
+            csv_row: Dict[str, Any],
+            timestamp_format: str = '%Y-%m-%d %H:%M:%S',
+    ) -> None:
+        """Consume Rotki generic trades import CSV file.
+        May raise:
+        - DeserializationError
+        - UnknownAsset
+        - KeyError
+        """
+        fee = Fee(deserialize_asset_amount(csv_row['Fee'])) if csv_row['Fee'] else None  # noqa: E501
+        fee_currency = (
+            symbol_to_asset_or_token(csv_row['Fee Currency'])
+            if csv_row['Fee Currency'] and fee is not None else None
+        )
+        amount_sold = deserialize_asset_amount(csv_row['Sell Amount'])
+        amount_bought = deserialize_asset_amount(csv_row['Buy Amount'])
+        trade = Trade(
+            timestamp=deserialize_timestamp_from_date(
+                date=csv_row['Timestamp'],
+                formatstr=timestamp_format,
+                location='Rotki generic trades importer',
+            ),
+            location=Location.EXTERNAL,
+            fee=fee,
+            fee_currency=fee_currency,
+            rate=Price(amount_sold / amount_bought),
+            base_asset=symbol_to_asset_or_token(csv_row['Base Currency']),
+            quote_asset=symbol_to_asset_or_token(csv_row['Quote Currency']),
+            trade_type=TradeType.SELL if csv_row['Type'] == 'Sell' else TradeType.BUY,
+            amount=amount_bought,
+            link=csv_row['ID'],
+            notes=csv_row.get('Description'),
```

Comment:
Don't mix `.get` and `[]` for accessing dict entries . If the row doesn't have a note it value will be `None` already right?